code/script.js

```diff
@@ -15,37 +207,113 @@ const showMessage = (message, sender) => {
+`;
+
+  showMessage(`Do you want regular coffee or chocolate?`, 'bot');
+  reply2CoffeeOrChoc();
+  removeLoading();
+};
+
+//Bot question 3 - Topping (For hot chocolate)
+const question3Topping = () => {
+  inputWrapper.innerHTML = `
+<select name="topping" id="topping">
+<option value="marschmallow topping">Marscmallow topping</option>
+<option value="chocolate cookie crumb topping">Chocolate Cookie Crumb topping</option>
+<option value="deluxe topping">Deluxe topping</option>
+<option value="no topping">No topping</option>
+</select>
+<button id="topping-next">Next</button>`;
+  showMessage(`Maybe with some topping on?`, 'bot');
+  reply3Topping();
+  removeLoading();
+};
+
+//Bot question 3 - Flavors (For Coffee)
+const question3Flavors = () => {
+  inputWrapper.innerHTML = `
+  <div id="flavors">
+  <form class="checkbox-form">
+    <div>
+    <input type="checkbox" class="flavors-boxes" id="vanilla" name="flavors" value="Vanilla" />
+    <label for="vanilla" class="textbox-label">Vanilla</label>
+    </div>
+    <div>
+    <input type="checkbox" class="flavors-boxes" id="caramel" name="flavors" value="Caramel"/>
+    <label for="caramel" class="textbox-label">Caramel</label>
+    </div>
+    <div>
+    <input type="checkbox" class="flavors-boxes" id="chai" name="flavors" value="chai"/>
+    <label for="chai" class="textbox-label">Chai</label>
+    </div>
+    <button id="flavors-next" type="submit">Next</button> 
+    </form>
+  </div>
+`;
+  showMessage(`What flavors would you like?`, 'bot');
+  reply3Flavors();
+  removeLoading();
+};
 
-// Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
+//Bot question 4 - Phone Number
+const question4PhoneNo = () => {
+  inputWrapper.innerHTML = `
+  <form id="phone-form">
+  <input type="tel" class="phone-number" name="phone" maxlength="12" pattern="[0-9]{3}[0-9]{3}[0-9]{4}" required> 
+  <button type="submit" id="confirm-btn">Confirm</button>
+  <button id="cancel-btn" onClick="window.location.reload();">Cancel</button>
+  </form>
+  `;
 
-// Set up your eventlisteners here
+  customerOrder.presentOrder();
+  showMessage(
+    `Please type in your phone number below and we'll notify you as soon as your ${customerOrder.type} is ready.`,
+    'bot'
+  );
+  reply4PhoneNo();
+  removeLoading();
+};
 
 // When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greetUser, 1000);
+setTimeout(question1Greeting, 500);
```

Comment:
Overall a really good job! I know it's not fully functional yet but you have a lot of code and it's only week 1 so you'll get there. I really like the loading animation and your code is structured nicely😃👏